tests/linux_kernel/test_stack_trace.py

```diff
@@ -94,3 +99,17 @@ def test_stack__repr_pretty_(self):
             assertReprPrettyEqualsStr(trace)
             for frame in trace:
                 assertReprPrettyEqualsStr(frame)
+
+    @skip_unless_have_test_kmod
+    def test_stack_locals(self):
+        task = self.prog["drgn_kthread"]
+        stack_trace = self.prog.stack_trace(task)
+        for frame in stack_trace:
+            if frame.symbol().name == "drgn_kthread_fn":
+                self.assertSetEqual(
+                    set(["arg", "a", "b", "c"]),
```

Comment:
Use a set literal.

```suggestion
                    {"arg", "a", "b", "c"},
```

Reply:
I truly wrote too much python 2.6-compatible code in a past life...